docs/specification/1.0.md

```diff
@@ -473,6 +473,7 @@ Field | Optionality | Type | Description
 ----- | ----- | ----- | --------
 `label` | REQUIRED | *string* | Human-readable label to display for this suggestion (e.g. the CDS Client might render this as the text on a button tied to this suggestion).
 `uuid` | OPTIONAL | *string* | Unique identifier, used for auditing and logging suggestions.
+`preference` | OPTIONAL | *string* | Allows a service to indicate how preferable this suggestion is from among all the available suggestions on the card. Allowed values are: `recommended`, and `alternate`. CDS Hooks clients may choose to influence their UI based on this value, such as pre-selecting recommended suggestions etc.
```

Comment:
Thanks for tagging me here @mattvarghese !

I like this concept and I'm trying to wrap my head around the model. For instance, why not just add an `isPreferred` boolean field? A few questions to explore in this vein...

1. Is the intention to add more values to the `{recommended, alternate}` enum over time? 
2. Is it allowable to mark more than one suggestion as `recommended`?
3. Is it allowable to mark all suggestions as `alternate`?